doc/date.md

```diff
@@ -0,0 +1,61 @@
+# Date
+
+`Napi::Date` class is a representation of the JavaScript `Date` object. The
+`Napi::Date` class inherits its behavior from `Napi::Value` class
+(for more info see [`Napi::Value`](value.md))
+
+## Methods
+
+### Constructor
+
+Creates a new _empty_ instance of a `Napi::Date` object.
+
+```cpp
+Napi::Date();
```

Comment:
Please change `Napi::Date()` to `Napi::Date::Date()`

Reply:
Yep, you're right.
These need fixed too
https://github.com/nodejs/node-addon-api/blame/master/doc/bigint.md#L42
https://github.com/nodejs/node-addon-api/blame/master/doc/number.md#L14
https://github.com/nodejs/node-addon-api/blob/master/doc/dataview.md
https://github.com/nodejs/node-addon-api/blob/master/doc/object_wrap.md
https://github.com/nodejs/node-addon-api/blob/master/doc/async_worker.md